### PR TITLE
templates: document how to turn on the new node styles

### PR DESCRIPTION
**Pull request by ilyagr**
Documents https://github.com/martinvonz/jj/pull/3501, though only in `templates.toml` for now (since this is mostly for jj developers to experiment with for now).

Let me know if you prefer that I document this in config.md.

### CODE REVIEW

**Comment by ilyagr** [comment]
I realized this conflicts with #3513. @yuja, I can close this if you add something like this to your PR, or I'll adjust this PR after yours gets merged.

File: cli/src/config/templates.toml [hunk 1]
```diff
@@ -166,6 +166,12 @@ if(hidden,
 )
 '''
 
+# Uncomment the following or put it in your config to enable facier graph node
+# styling.
+#
+# log_node = 'builtin_log_node'
+# op_log_node = 'builtin_op_log_node'
+
```
Comment (yuja) [inline cli/src/config/templates.toml:174]: I expect the current state is temporary, and the builtin node templates will be enabled in the next release.

> Let me know if you prefer that I document this in config.md.

Perhaps, some generalized doc can be added to `templates.md`? Something like
* to override the default, set `templates.<name>` where `<name>` is usually a command name
* e.g. `templates.log = 'builtin_log_comfortable'`
* stock templates are usually named as `builtin_..`
* the list can be obtained by `jj log -T` (without template value)

Comment (ilyagr) [reply cli/src/config/templates.toml:174]: This sounds like a good idea. I'll look into it.

This PR was meant for the short term, but I'm fine with not merging it; it should serve it's audience even unmerged.

> the list can be obtained by `jj log -T` (without template value)

That's really nice, good to know!

**Comment by ilyagr** [comment]
Cc #4095 that will hopefully replace this.

**Comment by algmyr** [comment]
> Cc #4095 that will hopefully replace this.

Yep, #4095 is now submitted and this is now obsolete, unless we still want to document these builtins somewhere. Will close this for now, we can re-open if we want some documentation still.

**Pull request closed without merging**